src/markdown/directives/parse.js

```diff
@@ -73,7 +76,18 @@ function parse(md, opts = {}) {
       let recognized = false
 
       for (const key of Object.keys(obj)) {
-        const globalKey = key.startsWith('$') ? key.slice(1) : key
+        const globalKey = key.startsWith('$')
+          ? (() => {
+              if (marpit.customDirectives.global[key]) return key
```

Comment:
We use an original directive key without warning if the dollar-prefixed custom global directive was defined.